### PR TITLE
Projects Page Update

### PR DESCRIPTION
**Pull request by hertelc**
Removed Old OSL projects and added link to CS312.  Tied to Issue #49

### CODE REVIEW

**Comment by hertelc** [comment]
@Kennric @kelnera @ramereth  - I'd love some feedback on my work!

**Comment by ramereth** [comment]
@hertelc I think you need to rebase your branch against the latest staging. Do you want to try and take a stab at that or would you like me to do it for you? You shouldn't have all those "Merge pull" commits included.

**Comment by kelnera** [comment]
@ramereth I had a similar issue a while back that @Kennric helped me work through. I don't know if it's the same thing, but I think it turned out to be okay.

**Comment by hertelc** [comment]
If you haven't done it I'd like to give it a try.  I'll see what I can do tonight.  I haven't rebased before, but I'll see what I can dig up and figure out.  Thanks for the opportunity!  It's good stuff for me to learn.

On Jul 29, 2016 3:07 PM, Lance Albertson <notifications@github.com> wrote:

@hertelc<https://github.com/hertelc> I think you need to rebase your branch against the latest staging. Do you want to try and take a stab at that or would you like me to do it for you? You shouldn't have all those "Merge pull" commits included.

**Comment by hertelc** [comment]
@ramereth, Ok I need some help. :)  I tried doing a rebase and I "think" it went well, but I'm not sure if it updated the pull request or what.   Here is the page I used: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request   I'm a little concerned I pushed it onto Master???  Thanks!

**Comment by ramereth** [comment]
@hertelc looks like you ran into some conflicts. When are you on campus this week? I forgot to stop by yesterday to help.

**Comment by hertelc** [comment]
I’m back on Thursday. Thanks for taking a look.

Carrie Hertel, PMP
Software Dev Group & Test Lab Director
Center for Applied Systems & Software
Oregon State University

**Comment by hertelc** [comment]
@kelnera - I'm ready for some +1's Finally!

**Comment by kelnera** [comment]
Why did the menu line in the students page get changed back to Organization? It should say People, Students, 2.

File: content/pages/organization/projects.rst [hunk 1]
```diff
@@ -130,17 +67,10 @@ OSL is pioneering the use of Linux and advanced configuration management on
...
-
-Supercell is OSL's Ganeti-based open compute and testing cluster, providing the
-open source community access to large virtual machine networks for building and
-testing software.
-
 CS 312
 ------
 
-CS 312, System Administration, is a hands-on course in modern methods of
+[CS 312](http://cs312.osuosl.org), System Administration, is a hands-on course in modern methods of
```
Comment (kelnera) [inline content/pages/organization/projects.rst]: This needs to wrap at 80 characters.

**Pull request merged**